### PR TITLE
Ml 139 pip after clicking the link

### PR DESCRIPTION
**Pull request by pkrucz00**
Functionality for automatic display of the main stream in the "picture-in-picture" format when the user clicks the product link.
Works on chrome. Does *not* work on safari.

### CODE REVIEW

File: assets/js/components/event/StreamArea.tsx [hunk 1]
```diff
@@ -41,16 +41,15 @@ const StreamArea = ({
...
   const [amIPresenter, setAmIPresenter] = useState<boolean>(false);
   const [presenterName, setPresenterName] = useState<string>("");
-  const { attachVideo, setSrc } = useHls(true, config);
-  const { device, orientation } = useContext(ScreenTypeContext);
+  const screenType = useContext(ScreenTypeContext);
   const { setStreamStart } = useStartStream();
   const [card, setCard] = useState<CardStatus>("hidden");
-  const showMobileBottomBar = device === "mobile" || orientation === "portrait";
-
-  useAutoHideMobileBottomBar(setCard);
```
Comment (kamil-stasiak) [inline assets/js/components/event/StreamArea.tsx:53]: Did you remove / revert this code accidentally or on purpose?

Comment (pkrucz00) [reply assets/js/components/event/StreamArea.tsx:53]: By accident. I'll include that back in

Comment (pkrucz00) [reply assets/js/components/event/StreamArea.tsx]: I've mended it.

File: assets/js/components/event/MobileBottomPanel.tsx [hunk 2]
```diff
@@ -28,9 +32,12 @@ export const MobileBottomPanel = ({
   isChatLoaded,
   isBannedFromChat,
   eventTitle,
+  enablePictureInPicture,
 }: Props) => {
+  const { device } = useContext(ScreenTypeContext);
   const isOpen = card !== "hidden";
 
+  if (device !== "mobile") return null;
```
Comment (kamil-stasiak) [inline assets/js/components/event/MobileBottomPanel.tsx:40]: I think `const { device } = useContext(ScreenTypeContext);` and `if (device !== "mobile") return null;` are here by mistake. We could remove them

File: assets/js/components/event/StreamArea.tsx [hunk 3]
```diff
@@ -101,7 +106,7 @@ const StreamArea = ({
         />
       )}
       <div className="Stream">
-        {mode === "hls" && (
+        {mode == "hls" && (
```
Comment (kamil-stasiak) [inline assets/js/components/event/StreamArea.tsx:109]: It would be safer to always use ===

File: assets/js/components/event/StreamArea.tsx [hunk 4]
```diff
@@ -41,10 +43,13 @@ const StreamArea = ({
...
   isBannedFromChat,
+  attachVideo,
+  setSrc,
+  enablePictureInPicture,
 }: StreamAreaProps) => {
   const [amIPresenter, setAmIPresenter] = useState<boolean>(false);
   const [presenterName, setPresenterName] = useState<string>("");
-  const { attachVideo, setSrc } = useHls(true, config);
+  const screenType = useContext(ScreenTypeContext);
   const { device, orientation } = useContext(ScreenTypeContext);
```
Comment (kamil-stasiak) [inline assets/js/components/event/StreamArea.tsx:53]: There are two `useContext(ScreenTypeContext)`. Please remove one of them

**Review by kamil-stasiak — CHANGES_REQUESTED** [review]

**Review by kamil-stasiak — APPROVED** [review]

**Pull request merged**